osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerMatchSongSelect.cs

```diff
@@ -19,8 +20,13 @@ public class MultiplayerMatchSongSelect : OnlinePlaySongSelect
         [Resolved]
         private MultiplayerClient client { get; set; }
 
+        [Resolved]
+        private Room room { get; set; }
+
         private LoadingLayer loadingLayer;
 
+        protected override UserActivity InitialActivity => new UserActivity.InLobby(room);
```

Comment:
This is not done for `PlaylistsSongSelect` which seems inconsistent.

In fact this should *probably* be moved to `OnlinePlaySongSelect`.

Reply:
Would be good if we changed `UserActivity.InLobby.Status` to "In a lobby" instead of "In a multiplayer lobby" perhaps?

Reply:
I suppose so.